src/manage-authors/add-coauthor.js

```diff
@@ -0,0 +1,17 @@
+const { gitAuthors } = require('../git-authors');
+
+async function addCoauthor(args) {
+  const coauthors = gitAuthors();
```

Comment:
You can destruct the array to make it more readable `addCoauthors([key, name, email])`